### ISSUE TITLE
[suggestion] Windowed mode

### ISSUE THREAD

**Issue by Podders** [codygibb/animus-visualizer#6]
Would it be possible to allow a chromeless Windowed setting? Would be great to be able to adjust the output resolution to something small enough to use for live streaming visualisation while being able to use/see other open windows

**Comment by dallasbpeters** [comment]
👍

**Comment by torinmb** [comment]
Yeah if you clone this repository and run the application using the
Processing IDE it default opens up in a windowed mode. After you install
Processing you’ll need to install the addons for minim, controlP5 and
syphon. Then open Animus.pde in the editor and click run.

Hope this helps!
